include/openspace/network/parallelconnection.h

```diff
@@ -83,6 +84,7 @@ class ParallelConnection  {
 
     ParallelConnection::Message receiveMessage();
 
+    static unsigned int protocolVersion();
```

Comment:
How about making the anonymous namespace variable a class variable instead and not have the indirection through this function?